agenta-web/package-lock.json

```diff
@@ -6896,8 +6896,9 @@
             }
         },
         "node_modules/prettier": {
-            "version": "3.0.2",
-            "integrity": "sha512-o2YR9qtniXvwEZlOKbveKfDQVyqxbEIWn48Z8m3ZJjBjcCmUy3xZGIv+7AkaeuaTr6yPXJjwv07ZWlsWbEy1rQ==",
```

Comment:
Do i need to revert the changes as of remote repository

Reply:
yes, this pr should not have any changes in package-lock.json file. One simple thing you can do is just copy the package-lock.json file contents from main branch and update your PR. Let me know if you need any help.
@Dhaneshwarguiyan